tests/testthat/test_OS_key.R

```diff
@@ -0,0 +1,20 @@
+context("test api key")
+
+test_that("multiplication works", {
+  expect_equal(2 * 2, 4)
+})
+
+
+
+test_that("set_api_key()", {
+						expect_error(set_os_key())
+#						expect_false(set_os_key(""))
+						expect_true(set_os_key(Sys.getenv("API_KEY")))
+})
+
+# Add:   GET and check status code
+test_that("get_os_key()", {
+						expect_true(is.character(Sys.getenv("API_KEY")	))
```

Comment:
I'd test `get_os_key()` and not `Sys.getenv()` since the latter is not part of the package.

Reply:
right.